delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/eventlog/GlobalEventLog.scala

```diff
@@ -13,23 +14,34 @@ trait GlobalEventLog[T] {
   /**
     * Get stream of all [[Event]]s
     * @param offset the offset to start from
+    * @param tag    optional used to filter events.
     */
-  def stream(offset: Offset): Stream[Task, T]
+  def stream(offset: Offset, tag: Option[TagLabel] = None): Stream[Task, Message[T]]
```

Comment:
Do we want the return type to be always a `Message[T]` ?
For SSEs we are using `Envelope[T]` instead.